sys/Makefile

```diff
@@ -10,6 +10,21 @@ endif
 ifneq (,$(filter shell_commands,$(USEMODULE)))
     DIRS += shell/commands
 endif
+ifneq (,$(filter slist,$(USEMODULE)))
```

Comment:
IMHO all following lines (excluding the olsr specific ones) are unnecessary in this file